### PR TITLE
Implemented static UI of select product screen (#1x69ahw)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
select-product
select-product-csv-upload

File: src/App.vue [hunk 1]
```diff
@@ -1,16 +1,35 @@
 <template>
   <ion-app>
-    <ion-router-outlet />
+    <ion-menu side="start" menu-id="first" content-id="main">
```
Comment (dt2patel) [inline src/App.vue:3]: Need to internationalize this page

File: src/views/SelectProductCSVUpload.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button slot="start" default-href="/" />
+        <ion-title>{{ $t("Upload CSV") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div class="upload-csv">
```
Comment (dt2patel) [inline src/views/SelectProductCSVUpload.vue:11]: use semantic tag here and update css selector

File: src/views/SelectProduct.vue [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+.section-grid {
+  grid-template-columns: repeat(auto-fill, 200px);
+}
+
+@media (min-width: 991px) {
+  .action {
+    /* desktop-only class is setting display to unset and below properties were not getting applied that is why display is set block here */
+    display: block;
+    /* TODO use spacer variable */
+    margin: 250px 0;
```
Comment (dt2patel) [inline src/views/SelectProduct.vue:168]: instead of doing this we should add a position of fixed here and position it at the bottom center

**Review by dt2patel — CHANGES_REQUESTED** [review]
Changes to css

File: src/App.vue (hunk 1 above)
Comment (azkyakhan) [reply src/App.vue:3]: Done

File: src/views/SelectProductCSVUpload.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/SelectProductCSVUpload.vue]: Done

File: src/views/SelectProduct.vue (hunk 3 above)
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: Done

**Review by dt2patel — CHANGES_REQUESTED** [review]
See comments

File: src/views/SelectProduct.vue [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+}
+.section-grid {
+  grid-template-columns: repeat(auto-fill, 200px);
+}
+
+@media (min-width: 991px) {
+  .action {
+    position: fixed;
+    bottom: 100px;
+    left: 850px;
```
[the remark this answers is not in the document]
Comment (azkyakhan) [reply src/views/SelectProduct.vue]: Done sir
action-button

**Review by dt2patel — APPROVED** [review]

**Pull request merged**